ang/civicase-base/providers/activity-forms.provider.js

```diff
@@ -0,0 +1,74 @@
+(function (_, angular) {
+  var module = angular.module('civicase-base');
+
+  module.provider('ActivityForms', ActivityFormsProvider);
+
+  /**
+   * Activity Forms Provider.
+   */
+  function ActivityFormsProvider () {
+    var activityForms = [];
+
+    this.$get = $get;
+    this.addActivityForms = addActivityForms;
+
+    /**
+     * Returns a `getActivityFormService` function that can be used to find
+     * right right form service for the given activity.
```

Comment:
Typo

Reply:
2 rights don't make a wrong! or was it the other way around? 🤔